package-lock.json

```diff
@@ -1,8 +1,7126 @@
 {
   "name": "validata",
   "version": "4.0.14",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Can you please rerun the installs with npm 6.x?
It might worth to upgrade it, but in an other PR.

Reply:
I reverted to the original package-lock.json, then re-installed with npm v6.14.15. Let me know if that looks ok.